packages/narada/src/narada/render_html.py

```diff
@@ -9,9 +8,5 @@ def render_html(html: str) -> None:
     Args:
         html: The HTML content to render.
     """
-    # Encode HTML to base64 for data URL
-    html_bytes = html.encode("utf-8")
```

Comment:
we didn't need base64 in the first place, base64 is only useful for encoding binary data (like images), but HTML is text so this strictly inflates the URL size